core/templates/pages/topic-viewer-page/practice-tab/practice-tab.directive.ts

```diff
@@ -43,12 +43,43 @@ angular.module('oppia').directive('practiceTab', [
           var ctrl = this;
 
           ctrl.newPracticeSession = function() {
+            var commaSeparatedSelectedSubtopics = '';
+            for (var idx in ctrl.selectedSubtopicIndices) {
+              if (ctrl.selectedSubtopicIndices[idx]) {
+                commaSeparatedSelectedSubtopics += (
+                  ctrl.availableSubtopics[idx].getTitle() + ',');
+              }
+            }
             var practiceSessionsUrl = UrlInterpolationService.interpolateUrl(
               PRACTICE_SESSIONS_URL, {
-                topic_name: ctrl.getTopicName()
+                topic_name: ctrl.getTopicName(),
+                // These are actually the selected subtopics, but named so
+                // because this will be shown in the URL and for the learner,
```

Comment:
I think it's ok to still call them subtopics; learners won't pay attention to long urls that much.

Reply:
Done